React/ReactRedux/What are the drawbacks of MVW pattern.md

```diff
@@ -0,0 +1,7 @@
+## What are the drawbacks of MVW pattern?
```

Comment:
Please, do not use first google search result as a question answer. It is incorrect answers. How MVP pattern might affect DOM manipulation?